packages/react-lowcode/src/codegen/ast/widgetDeclaration.ts

```diff
@@ -100,4 +126,43 @@ function isTableDeclaration(node: ts.Node): boolean{
 function isDetailDeclaration(node: ts.Node): boolean{
     ///TODO: check also for export key word
     return ts.isVariableDeclaration(node)
+}
+
+function isElementWithAttributes(node: ts.Node): node is ts.JsxOpeningElement | ts.JsxSelfClosingElement {
+    return (ts.isJsxOpeningElement(node) || ts.isJsxSelfClosingElement(node)) && node.attributes.properties.length > 0;
+}
+
+function getAttributeValue(attribute: ts.JsxAttribute): string | undefined {
```

Comment:
please simplify function body like this:
```
    if (attribute.initializer) {
        return getStringTypeAttributeValue(attribute.initializer)
            || getNumberTypeAttributeValue(attribute.initializer)
            || getBooleanTypeAttributeValue(attribute.initializer);
    }

   return 'true';
```
In your future code:
* no need to else return at the end of the function
* no need to define one return variable modified by multiple branches